TarponSimulator2017/Controller/SceneInGame.cs

```diff
@@ -13,13 +17,19 @@ public sealed class SceneInGame : Scene
 
 		private SceneInGame()
 		{
-			//actions = new Dictionary<Keys,Command>();
+			actions = new Dictionary<Keys,Command>();
 			//this.actions.Add(Keys.Up, new CommandAccelerate(world.playerBoat));
 			//this.actions.Add(Keys.Right, new CommandTurn (world.playerBoat));
 			//this.actions.Add(Keys.Left, new CommandTurn (world.playerBoat));
 		}
 
-
+		public SceneInGame(World world){
+			actions = new Dictionary<Keys,Command> ();
+			Scene.world = world;
+			this.actions.Add (Keys.Up, new CommandAccelerate (world.playerBoat));
```

Comment:
Well, we'll have to find a linter / format tool.
As here you have a space between Add and parenthesis but not in the 2 following lines.